vitrina/datasets/models.py

```diff
@@ -121,6 +121,27 @@ def get_tag_list(self):
         return str(self.tags).replace(" ", "").split(',') if self.tags else []
 
 
+class DatasetMembers(models.Model):
```

Comment:
```suggestion
class DatasetMember(models.Model):
```

Model in Django are named in singular form.